esptfa_arima/Authentication/services.py

```
@@ -148,45 +207,68 @@ def get_sections_dict() -> dict:
-                    lrn=lrn,
-                    first_name=first_name,
-                    middle_name=middle_name,
-                    last_name=last_name,
-                    section=section,
-                    user_id=None
+            try:
+                lrn = str(student.get("lrn")).strip()
+                if len(lrn) != 11:
+                    raise DRFValidationError(
+                        f"Row {index + 1}:LRN '{lrn}' is not 11 characters long"
+                    )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)